### PR TITLE
Code Formatting Using Patched ClangFormat

### PR DESCRIPTION
**Pull request by TheBrokenRail**
If there is any issues with the formatting (which there probably are), please comment!

### CODE REVIEW

File: AppKit/CoreData_/NSPersistentDocument.m [hunk 1]
```diff
@@ -1,116 +1,114 @@
...
     NSUnimplementedMethod();
     return NO;
 }
 
-
 - (BOOL) revertToContentsOfURL: (NSURL *) inAbsoluteURL
                         ofType: (NSString *) inTypeName
-                         error: (NSError **) outError
-{
+                         error: (NSError **) outError {
```
Comment (bugaevc) [inline AppKit/CoreData_/NSPersistentDocument.m:102]: This is the case I was talking about: if the selector is multi-line, I prefer to put the brace on a separate line (as it was here).

Comment (TheBrokenRail) [reply AppKit/CoreData_/NSPersistentDocument.m:102]: I don't think ClangFormat supports that, its either attached or not-attached.

Comment (bugaevc) [reply AppKit/CoreData_/NSPersistentDocument.m:102]: You're already patching clang-format, aren't you :D?

Comment (TheBrokenRail) [reply AppKit/CoreData_/NSPersistentDocument.m:102]: I'm waiting for LLVM to compile to see if the patch worked.

Comment (TheBrokenRail) [reply AppKit/CoreData_/NSPersistentDocument.m]: Patched!

**Comment by TheBrokenRail** [comment]
Any Updates?

**Comment by bugaevc** [comment]
I'm just going to push the big button (hides)

**Comment by bugaevc** [comment]
Seriously though, thanks a lot for your work! I was looking forward to reformatting Cocotron for a long time 😀

**Pull request merged**